pulp_service/pulp_service/app/viewsets.py

```diff
@@ -138,3 +141,27 @@ def get_queryset(self):
     @classmethod
     def view_name(cls):
         return "admintasks"
+
+
+class ContentScan(APIView):
+    authentication_classes = []
+    permission_classes = []
+
+    def post(self, request=None):
+        serialized_data = ContentScanSerializer(data=request.data)
+        serialized_data.is_valid(raise_exception=True)
+        repo_version_pk = serialized_data.data["repo_version"]
+        task = dispatch(check_content, kwargs={"repo_version_pk": repo_version_pk})
+        return Response(task.pk)
```

Comment:
we should return an OperationPostponedResponse here similar to https://github.com/pulp/pulpcore/blob/main/pulpcore/app/viewsets/reclaim.py#L53